src/internal/pjs/server_test.go

```diff
@@ -149,14 +272,14 @@ func TestWalkJob(t *testing.T) {
 
 // runJob does work through PJS.
 func runJob(t *testing.T, ctx context.Context, c pjs.APIClient, in *pjs.CreateJobRequest,
-	fn func(resp *pjs.ProcessQueueResponse) error) (*pjs.JobInfo_Success, error) {
+	fn func(resp *pjs.ProcessQueueResponse) error, afterDequeue func()) (*pjs.JobInfo_Success, error) {
 	jres, err := c.CreateJob(ctx, in)
 	require.NoError(t, err)
-	return runJobFrom(t, ctx, c, jres.Id.Id, in.Program, fn)
+	return runJobFrom(t, ctx, c, jres.Id.Id, in.Program, fn, afterDequeue)
 }
 
 func runJobFrom(t *testing.T, ctx context.Context, c pjs.APIClient, from int64, programStr string,
-	fn func(resp *pjs.ProcessQueueResponse) error) (*pjs.JobInfo_Success, error) {
+	fn func(resp *pjs.ProcessQueueResponse) error, afterDequeue func()) (*pjs.JobInfo_Success, error) {
```

Comment:
I think we should change `fn` to some name that `fn` encloses what the job does.
I'm not sure what the best name is? do(), process(), et cetera. I think having a more descriptive name helps explain what it does versus afterDequeue()

Reply:
I'll remove this parameter